code/modules/events/sentience.dm

```diff
@@ -36,9 +36,9 @@
 	SA.universal_speak = 1
 	SA.sentience_act()
 	SA.can_collar = 1
-	SA.maxHealth = max(SA.maxHealth + 200)
-	SA.melee_damage_lower = max(SA.melee_damage_lower + 15)
-	SA.melee_damage_upper = max(SA.melee_damage_upper + 15)
+	SA.maxHealth = SA.maxHealth + 200
```

Comment:
Can also change these to SA.val += number
Instead of SA.val = SA.val + number

Reply:
Good call.